### PR TITLE
PIP 11 cards

### PR DESCRIPTION
**Pull request by Simisays**
Some of these were quite the headache

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+PT:3/3
+K:Reach
+K:Lifelink
+Reach, lifelink
+T:Mode$ AttackersDeclared | Execute$ TrigPutCounter | ValidAttackers$ Creature | ValidAttackersAmount$ GE2 | TriggerZones$ Battlefield | AttackingPlayer$ You | TriggerDescription$ One for My Baby — Whenever you attack with two or more creatures, put two quest counters on CARDNAME. When you do, NICKNAME deals damage equal to the number of quest counters on it to up to one target creature unless that creature's controller has CARDNAME deal that much damage to them.
+SVar:TrigPutCounter:DB$ PutCounter | CounterType$ QUEST | CounterNum$ 2 | RememberPut$ True  | SubAbility$ DBImmediateTrigger 
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | Execute$ TrigDealDamage | SpellDescription$ When you do, NICKNAME deals damage equal to the number of quest counters on it to up to one target creature unless that creature's controller has CARDNAME deal that much damage to them.
+SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Creature | TgtPrompt$ Select up to one target creature | TargetMin$ 0 | TargetMax$ 1 | NumDmg$ X | UnlessCost$ DamageYou<X> | UnlessPayer$ TargetedController
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt:11]: Seems like it recognizes the correct amount (with the prompt saying do you want to deal X to targeted) but it doesn't seem to deal damage on resolution

File: forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Cait, Cage Brawler
+ManaCost:R G
+Types:Legendary Creature Human Warrior
+PT:1/1
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Indestructible | Condition$ PlayerTurn | Description$ As long as it's your turn, CARDNAME has indestructible.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever NICKNAME attacks, you and defending player each draw a card, then discard a card. Put two +1/+1 counters on NICKNAME if you discarded the card with the highest mana value among those cards or tied for highest.
+SVar:TrigDraw:DB$ Draw | Defined$ You & TriggeredDefender | NumCards$ 1 | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Mode$ TgtChoose | Defined$ You & TriggeredDefender | RememberDiscarded$ True | SubAbility$ FindHighestCMC
+SVar:FindHighestCMC:DB$ Pump | ImprintCards$ ValidAll Card.greatestRememberedCMC | SubAbility$ DBPutCounter
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt:9]: This line might be unnecesairy ?

File: forge-gui/res/cardsfolder/upcoming/diamond_city.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Diamond City
+ManaCost:
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/diamond_city.txt:2]: remove this line

File: forge-gui/res/cardsfolder/upcoming/cathedral_acolyte.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Cathedral Acolyte
+ManaCost:1 G
+Types:Creature Human Cleric
+PT:1/2
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+HasCounters | AddKeyword$ Ward:1 | Description$ Each creature you control with a counter on it has ward {1}. (Whenever it becomes the target of a spell or ability an opponent controls, counter it unless that player pays {1}.)
+A:AB$ PutCounter | Cost$ T | ValidTgts$ Creature.ThisTurnEntered | TgtPrompt$ Select target creature that entered the battlefield this turn | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on target creature that entered the battlefield this turn.
+DeckHas:Ability$Counter
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cathedral_acolyte.txt:7]: typos

File: forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Name:Craig Boone, Novac Guard
+ManaCost:1 R W
+Types:Legendary Creature Human Soldier
+PT:3/3
+K:Reach
+K:Lifelink
+Reach, lifelink
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt:7]: bad line

File: forge-gui/res/cardsfolder/upcoming/curie_emergent_intelligence.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Name:Curie, Emergent Intelligence
+ManaCost:1 U
+Types:Legendary Artifact Creature Robot
+PT:1/3
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, draw cards equal to its base power.
+SVar:TrigDraw:DB$ Draw | NumCards$ Count$CardBasePower | Defined$ You
+A:AB$ Clone | Cost$ 1 U Exile<1/Artifact.nonToken+Other+Creature/select another target artifact creature> | AddTriggers$ TrigDamage | AddSVars$ TrigDamage,TrigDrawBis | Defined$ Exiled | SpellDescription$ NICKNAME becomes a copy of the exiled creature, except it has "Whenever this creature deals combat damage to a player, draw cards equal to its base power."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/curie_emergent_intelligence.txt:7]: @Hanmac
which SVars still need adding?

File: forge-gui/res/cardsfolder/upcoming/desdemona_freedoms_edge.txt [hunk 7]
```diff
@@ -0,0 +1,10 @@
+Name:Desdemona, Freedom's Edge
+ManaCost:2 R W
+Types:Legendary Creature Human Rogue
+PT:3/4
+K:Vigilance
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks, target creature card in your graveyard that's an artifact or that has mana value 3 or less gains escape until end of turn. The escape cost is equal to its mana cost plus exile two other cards from your graveyard. (You may cast it from your graveyard for its escape cost this turn.)
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.Artifact+YouCtrl,Creature.cmcLE3+YouCtrl | TgtZone$ Graveyard | PumpZone$ Graveyard | TgtPrompt$ Select target creature card in your graveyard that's an artifact or that has mana value 3 or less | KW$ Escape:CardManaCost ExileFromGrave<3/Card.Other/other>
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/desdemona_freedoms_edge.txt:7]: wrong KW

File: forge-gui/res/cardsfolder/upcoming/diamond_city.txt [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+Types:Land
+ManaCost:no cost
+K:etbCounter:Shield:1
+A:AB$ Mana | Cost$ T | Produced$ C | SpellDescription$ Add {C}.
+A:AB$ MoveCounter | Cost$ T | Source$ Self | ValidTgts$ Creature | IsPresent$ Creature.YouCtrl+ThisTurnEntered | PresentCompare$ GE2 | CounterType$ Shield | CounterNum$ 1 | SpellDescription$ Move a shield counter from CARDNAME onto target creature. Activate only if two or more creatures entered the battlefield under your control this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/diamond_city.txt:7]: wrong condition

File: forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt [hunk 9]
```diff
@@ -0,0 +1,14 @@
+Name:Cait, Cage Brawler
+ManaCost:R G
+Types:Legendary Creature Human Warrior
+PT:1/1
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Indestructible | Condition$ PlayerTurn | Description$ As long as it's your turn, CARDNAME has indestructible.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever NICKNAME attacks, you and defending player each draw a card, then discard a card. Put two +1/+1 counters on NICKNAME if you discarded the card with the highest mana value among those cards or tied for highest.
+SVar:TrigDraw:DB$ Draw | Defined$ You & TriggeredDefender | NumCards$ 1 | SubAbility$ DBDiscard
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt:7]: wrong defined

File: forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+SVar:DBDiscard:DB$ Discard | Mode$ TgtChoose | Defined$ You & TriggeredDefender | RememberDiscarded$ True | SubAbility$ FindHighestCMC
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt:8]: discard does not support & for its defined syntax currently
use older way

File: forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt:9]: yes

File: forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+T:Mode$ AttackersDeclared | Execute$ TrigPutCounter | ValidAttackers$ Creature | ValidAttackersAmount$ GE2 | TriggerZones$ Battlefield | AttackingPlayer$ You | TriggerDescription$ One for My Baby — Whenever you attack with two or more creatures, put two quest counters on CARDNAME. When you do, NICKNAME deals damage equal to the number of quest counters on it to up to one target creature unless that creature's controller has CARDNAME deal that much damage to them.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt:8]: NICKNAME

File: forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt:11]: works for me?

File: forge-gui/res/cardsfolder/upcoming/curie_emergent_intelligence.txt (hunk 6 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/curie_emergent_intelligence.txt:7]: It does seem to crash without the SVar line

File: forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt (hunk 10 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt:8]: Using a repeatplayer effect ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt:8]: TriggeredDefendingPlayerAndYou

File: forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt (hunk 1 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt:10]: You're right actually

File: forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt (hunk 2 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt]: Hmm any thoughts on the variable for the condition ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt]: you should check `ConditionDefined$ Remembered` for the Valid

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt]: Should the conditionpresent$ contain GreatestCMC or greatestRememberedCMC ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt]: you'll want the CardProperty you've been using

File: forge-gui/res/cardsfolder/upcoming/colonel_autumn.txt [hunk 12]
```diff
@@ -0,0 +1,12 @@
+Name:Colonel Autumn
+ManaCost:1 W B
+Types:Legendary Creature Human Soldier
+PT:2/3
+K:Lifelink
+K:Exploit
+S:Mode$ Continuous | Affected$ Card.YouCtrl+Legendary+Other | AddKeyword$ Exploit | Description$ Other legendary creatures you control have exploit.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/colonel_autumn.txt:7]: wrong valid

**Review by tool4ever — APPROVED** [review]

**Pull request merged**